webui/react/src/pages/ExperimentList.table.tsx

```diff
@@ -49,6 +50,13 @@ export const columns: ColumnsType<ExperimentItem> = [
     sorter: (a: ExperimentItem, b: ExperimentItem): number => (a.progress || 0) - (b.progress || 0),
     title: 'Progress',
   },
+  {
+    align: 'center',
+    render: experimentArchivedRenderer,
+    sorter: (a: ExperimentItem, b: ExperimentItem): number =>
+      (a.archived === b.archived)? 0 : a.archived? 1 : -1,
```

Comment:
**nit:** nice, love the nested ternary. Don't have to this now but if we see another boolean based sorting use case, we can abstract into a dedicated `booleanSorter`. How about this for a little better legibility?
```
(a.archived === b.archived) ? 0 : (a.archived ? 1 : -1),
```